src/realm/list.hpp

```diff
@@ -656,6 +656,15 @@ inline T Lst<T>::get(size_t ndx) const
     if (ndx >= current_size) {
         throw std::out_of_range("Index out of range");
     }
+
+    // proxy out the mixed value for links, we need to know if the link is valid or not (not necesseraly must it be
+    // null)
+    if constexpr (std::is_same<T, Mixed>::value) {
+        Mixed mixed_link_value = m_tree->get(ndx);
+        if (mixed_link_value.is_null())
```

Comment:
I assume this should be `if (mixed_link_value.is_uressolved_link())`

Reply:
Yes, I wanted to call ```is_unresolved_link()```